package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stepfinance/solana-market-aggregator",
-  "version": "1.17.1",
+  "version": "1.17.2",
```

Comment:
In future don't bump the version in the same PR. There is another process outlined in the README which does the version tagging etc.